src/v2/Apps/Fair/Routes/FairBooths.tsx

```diff
@@ -0,0 +1,217 @@
+import { Media } from "v2/Utils/Responsive"
+import { usePrevious } from "v2/Utils/Hooks/usePrevious"
+import createLogger from "v2/Utils/logger"
+import { FairExhibitorRailFragmentContainer as FairExhibitorRail } from "../Components/FairExhibitorRail"
+import { FairExhibitorSortFilter } from "../Components/FairExhibitorSortFilter"
+import { PaginationFragmentContainer as Pagination } from "v2/Components/Pagination"
+
+const logger = createLogger("FairExhibitors.tsx")
+
+const PAGE_SIZE = 15
+
+interface FairBoothsProps {
+  fair: FairBooths_fair
+  relay: RelayRefetchProp
+}
+
+const FairBooths: React.FC<FairBoothsProps> = ({ fair, relay }) => {
+  const context = useExhibitorsFilterContext()
+
+  const [isLoading, setIsLoading] = useState(false)
+
+  const previousFilters = usePrevious(context.filters!)
+
+  const {
+    pageInfo: { hasNextPage },
+    pageCursors,
+  } = fair.exhibitors!
+
+  useDeepCompareEffect(() => {
+    const filtersHaveUpdated = Object.entries(context.filters!).some(
+      ([filterKey, currentFilter]) => {
+        const previousFilter = previousFilters[filterKey]
+        return !isEqual(currentFilter, previousFilter)
+      }
+    )
+
+    if (filtersHaveUpdated) {
+      fetchResults()
+    }
+  }, [context.filters])
+
+  function fetchResults() {
+    setIsLoading(true)
+
+    const relayParams = {
+      id: fair.slug,
+      first: PAGE_SIZE,
+    }
+
+    const relayRefetchVariables = {
+      ...relayParams,
+      ...context.filters,
+    }
+
+    relay.refetch(relayRefetchVariables, null, error => {
+      if (error) {
+        logger.error(error)
+      }
+
+      setIsLoading(false)
+    })
+  }
+
+  function loadPage(page) {
+    context.setFilter("page", page)
+  }
+
+  function loadNext() {
+    if (fair.exhibitors?.pageInfo.hasNextPage) {
+      loadPage(context?.filters?.page! + 1)
+    }
+  }
+
+  return (
+    <>
+      <Box id="jump--ExhibitorsFilter" />
+
+      <Media at="xs">
+        <Sticky>
+          {({ stuck }) => {
+            return (
+              <Flex
+                justifyContent="flex-end"
+                py={1}
+                {...(stuck
+                  ? {
+                      px: 2,
+                      borderBottom: "1px solid",
+                      borderColor: "black10",
+                    }
+                  : {})}
+              >
+                <FairExhibitorSortFilter />
+              </Flex>
+            )
+          }}
+        </Sticky>
+      </Media>
+
+      <Media greaterThan="xs">
+        <Flex justifyContent="flex-end">
+          <FairExhibitorSortFilter />
+        </Flex>
+      </Media>
+
+      <Spacer mt={4} />
+
+      <LoadingArea isLoading={isLoading}>
+        {fair.exhibitors?.edges!.map((edge, index) => {
+          const show = edge?.node!
+          if (show?.counts?.artworks === 0 || !show?.partner) {
+            // Skip rendering of booths without artworks
+            return null
```

Comment:
**question**: I realize this isn't new code but as a follow-up I wonder if we can use the `isDisplayable` field to determine whether we should render or not.